### DISCUSSION TITLE
No domains have been added for this server Sending via mailer

### DISCUSSION THREAD

**Discussion by Debra5750** [postalserver/postal#2319] [Help with using Postal]
What does this mean on postal? I thought everything was done but obviously not. Can someone help me please

**Comment by willpower232** [comment]
https://github.com/postalserver/postal/blob/2e7aaaba5a9a37f32bdff9a9fde0379834b2a4db/app/views/servers/_header.html.haml#L10

It is just some information, you should see a Domains tab where you can add one and set up the DNS records so that emails sent with Postal work.
